packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -453,6 +523,13 @@ export function PrescribeWorkflow(props: PrescribeProps) {
                       weightUnit={props.weightUnit}
                       prefillNotes={prefillNotes}
                       enableCombineAndDuplicate={props.enableCombineAndDuplicate}
+                      screenDraftedPrescriptions={function () {
+                        screenDraftedPrescriptions();
+                      }}
```

Comment:
i chose to do it this way in case we wanted to add anything else into these event hooks - would be easier to insert a line here than it would if we did anything else

but that's a lil bit premptive - there's nothing wrong with just doing `f` and it's not that hard to do other stuff later on if necessary